test/controllers/api/jobs_controller_test.rb

```diff
@@ -0,0 +1,96 @@
+require 'test_helper'
+
+class Api::JobsControllerTest < ActionController::TestCase
+
+  let(:user) { User.create!(email: 'test@prx.org', password: 'foobarpassword') }
+
+  let(:application) do
+    Doorkeeper::Application.create(
+      name: 'test',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.